src/wpasupplicant/Makefile

```diff
@@ -0,0 +1,36 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = wpasupplicant_$(WPASUPPLICANT_VERSION)_$(CONFIGURED_ARCH).deb
+DERIVED_TARGETS = wpasupplicant-dbgsym_$(WPASUPPLICANT_VERSION)_$(CONFIGURED_ARCH).deb
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Remove any stale files
+	rm -rf ./wpa
+
+	# Clone wpa repo
+#	git clone https://github.com/Azure/sonic-wpa-supplicant.git
```

Comment:
What's the issue regarding the repo in Azure? Why are we still using debian repo?

Reply:
Because the repo in Azure doesn't include debian rule to build the debian package.
This PR need wait another PR(haven't been created) to add debian build rule for sonic-wpa-supplicant. So this PR is draft status right now.